uarray/_backend.py

```diff
@@ -618,8 +703,13 @@ def determine_backend(value, dispatch_type, *, domain, only=True, coerce=False):
 
 
 def determine_backend_multi(
-    dispatchables, *, domain, only=True, coerce=False, **kwargs
-):
+    dispatchables: Iterable[Any],
+    *,
+    domain: str,
+    only: bool = True,
+    coerce: bool = False,
+    **kwargs: type[Any],
+) -> _SetBackendContext:
```

Comment:
You could overload here w.r.t. the `dispatch_type` keyword-only kwarg, but I'm not sure it's worth the extra `overload` clutter.

Reply:
It would be incorrect, as even with a default `dispatch_type`, the output types are arbitrary.